### PR TITLE
Phase2-TB74 Enable Simulation Runs for 6 inch wafers as well

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Enables Simulation Runs for 6-inch wafers as well. It was not working for the last several CMSSW cycles. This PR does not affect any Phase2 workflows that utilise the entire HGCal detector.

#### PR validation:

Tested using a cfg file in SimG4CMS/HGCalTestBeam/test

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

File: SimG4CMS/Calo/interface/HGCSD.h [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #include "SimG4Core/Notification/interface/BeginOfEvent.h"
 #include "SimG4CMS/Calo/interface/HGCNumberingScheme.h"
 #include "SimG4CMS/Calo/interface/HGCMouseBite.h"
+#include "Geometry/HGCalTBCommonData/interface/HGCalTBDDDConstants.h"
 
 #include <string>
 #include <TTree.h>
```
Comment (civanch) [inline SimG4CMS/Calo/interface/HGCSD.h:17]: @bsunanda , why this class depend on TTree?

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimG4CMS/Calo/interface/HGCSD.h (hunk 1 above)
Comment (bsunanda) [reply SimG4CMS/Calo/interface/HGCSD.h:20]: Because when we use plot debug we make a tree within the SD class

Comment (bsunanda) [reply SimG4CMS/Calo/interface/HGCSD.h:20]: @civanch @srimanob Please sign this PR

**Comment by srimanob** [comment]
Hi @bsunanda 
Would you mind to add what to expect from this PR in the PR description. From the title, I can't say this PR will effect any Phase-2 workflow or not. The PR test does not show the effect, so I assume it does not. But it is better to have a clear statement on what do you want from this PR. 

Regarding the current description:
"Enables Simulation Runs for 6-inch wafers as well. It was not working for last several CMSSW cycles"
Is this PR kind of bug-fix? Or maybe mentioning on why it does not work will help.

**Comment by bsunanda** [comment]
It is indeed a bug fix. When the codes for 6-inch and 8-inch wafers got separated, the code for 8-inch wafers was working fine. However, the code for 6-inch wafers, which is used for the test beam studies, needs this PR to complete the SIM step correctly.

**Comment by srimanob** [comment]
+Upgrade

**Comment by civanch** [comment]
+1

**Comment by antoniovilela** [comment]
+1

**Pull request merged**